charts/vela-core/templates/defwithtemplate/cpuhpa.yaml

```diff
@@ -38,8 +38,14 @@ spec:
         	}
         }
         parameter: {
-        	min:     *1 | int
-        	max:     *10 | int
+        
+        	// +usage=specify the min number of replicas to which the autoscaler can scale down
```

Comment:
Suggest not to use `min` as you are explaining `min` at the very moment.

Reply:
thanks! I will fix it.